ggshield/cmd/iac/scan/all.py

```diff
@@ -118,10 +123,11 @@ def display_iac_scan_all_result(
     directory: Path,
     result: Union[IaCScanResult, IaCSkipScanResult, None],
 ) -> int:
+    root = get_git_root(directory) if is_git_dir(directory) else directory.resolve()
```

Comment:
I think you can use your new `get_project_root_dir()` function here.

Reply:
Good catch thanks, in fact this is already done in the output handlers, so I've removed it here